### ISSUE TITLE
Issue in training the Code

### ISSUE THREAD

**Issue by avigupta2798** [JongSuk1/AVCap#2]
Thank you for your paper and for providing the code. There seems to be some issue while training the code. Even after training for 15 epochs, the training loss does not converge even by a small margin. Furthermore, the Validation loss is continuously coming to be 0. Also, all the metrics show a result of 0. Could you help in this regard?
I am training the same code on 2 4090 GPUs with the same batch size and another hyperparameter, as given in the README file.

**Comment by hjglhlha** [comment]
> Thank you for your paper and for providing the code. There seems to be some issue while training the code. Even after training for 15 epochs, the training loss does not converge even by a small margin. Furthermore, the Validation loss is continuously coming to be 0. Also, all the metrics show a result of 0. Could you help in this regard? I am training the same code on 2 4090 GPUs with the same batch size and another hyperparameter, as given in the README file.

I would like to ask you: Did you retrain the pre-trained model model_git_base.pt, or where can I download it directly? I did not see a place to download the model directly in https://github.com/microsoft/GenerativeImage2Text.
